ocs_ci/ocs/resources/noobaa.py

```diff
@@ -81,6 +81,17 @@ def s3_get_all_bucket_objects(self):
         """
         return [bucket for bucket in self._s3_resource.buckets.all()]
 
+    def s3_verify_bucket_exists(self, bucket):
+        """
+        Args:
+            bucket: The bucket object to be verified
+
+        """
```

Comment:
Shouldn't we return the result if the bucket exists or not?

Reply:
Yes, made the change.